components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -41,42 +55,82 @@ var credentialHelper = &cobra.Command{
 			if token == "" {
 				token = "no"
 			}
+			log.Infof("returning username=%s\npassword=%s\n", user, token)
```

Comment:
We should not leak the Git credentials into a log file here.

Reply:
Agreed